source/api/image/3.0/index.md

```diff
@@ -395,6 +395,8 @@ The `default` quality exists to support [level 0 compliant implementations][imag
 
 Regardless of level of compliance, services that make additional qualities beyond `default` available _MUST_ list those qualities in the [`extraQualities` property][image3-extra-functionality] of the response to the [Image Information Request][image3-information]. A request for an image with an unsupported quality value _SHOULD_ return a 400 (Bad Request) status code.
 
+**N.B.** A request for the `color` quality of an image may return an image comprised solely of grayscale or bitonal pixels if this is all of the color information the server has available. The server _SHOULD NOT_ include `color` in the `extraQualities` list when this is a case, but a request of this nature _MUST NOT_ result in an error.
```

Comment:
I think to be consistent with formatting we should use a note like "Model Alignment" in https://iiif.io/api/presentation/3.0/#331-external-links , along the lines of:

```
__Qualities Note__<br />
...stuff in here...
{: .note}
```

Reply:
Also, the last sentence is really hard to parse with two "this". Perhaps:

> The server _SHOULD NOT_ include `color` in the `extraQualities` list in such cases, but a request for the `color` quality _MUST NOT_ result in an error.

Should there also be:

> Similarly, a request for the `greyscale` quality may return an image comprised solely of bitonal pixels.